src/main/java/uk/gov/hmcts/reform/civil/model/sdo/ReferToJudgeReferences.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.civil.model.sdo;
```

Comment:
I don't think this class is needed, it is trying to encompass the two ccd fields but to my knowledge they fields themselves would be need to be fields in caseData as if you try to add this class to caseData, it will look for a ccd field with this name.

Alternatively, you can make the ccd fields inside a complexType with this name and it should then capture in caseData.